omnigibson/prims/joint_prim.py

```diff
@@ -793,8 +805,9 @@ def set_effort(self, effort, normalized=False):
                 de-normalized first before being executed). Default is False
         """
         # Sanity checks -- make sure that we're articulated (no control type check like position and velocity
-        # because we can't set effort targets)
+        # because we can't set effort targets) and that we're driven
         self.assert_articulated()
+        assert self._driven, "Cannot set efforts for joint that is not driven!"
```

Comment:
If it's not driven, can you set_vel and set_pos? Let's maybe discuss in person.

Reply:
changed `target` to `drive` for better clarity